### ISSUE TITLE
console.table display bug

### ISSUE THREAD

**Issue by dipamsen** [processing/p5.js-web-editor#1520]
#### Nature of issue?

- Found a bug

#### Details about the bug: 
`console.table` does not display properly in console window.

![image](https://user-images.githubusercontent.com/59444569/88699035-73336f80-d124-11ea-935a-7e1546213035.png)

**Comment by catarak** [comment]
Thanks for reporting!

**Comment by andrewragy** [comment]
I can't seem to find seem to find the console table jsx element.

**Comment by catarak** [comment]
@andrewragy console.table is rendered using the [console-feed](https://www.npmjs.com/package/console-feed) library!

**Comment by andrewragy** [comment]
I have a question: why doesn't the editor use the monacco editor, it's newer than codemirror and has a more modern ui

**Comment by ankiiitraj** [comment]
I found
In chromium -
![Screenshot from 2020-08-27 18-25-43](https://user-images.githubusercontent.com/48787278/91444822-ea2e6600-e892-11ea-820d-a31c6a56926b.png)

In FireFox -
![Screenshot from 2020-08-27 18-26-51](https://user-images.githubusercontent.com/48787278/91444916-07633480-e893-11ea-8d89-c33e4a8d042a.png)

**Comment by KKVANONYMOUS** [comment]
@catarak I would like to work on this issue. Can you please guide me on where to start?

**Comment by catarak** [comment]
@KKVANONYMOUS sure! I would take a look at the [console-feed](https://www.npmjs.com/package/console-feed) library and see what styling attributes you can change.

**Comment by KKVANONYMOUS** [comment]
> @KKVANONYMOUS sure! I would take a look at the `[console-feed](https://www.npmjs.com/package/console-feed)` library and see what styling attributes you can change.

@catarak I looked for a way to solve this issue in `console-feed` documentation but could not find anything useful. 
I tried changing certain Table related attributes, but nothing happened. 
If you know some way out of this, please let me know.

**Comment by catarak** [comment]
This issue might unfortunately take some digging into the console-feed code—my next approach would be to look into how console.table is implemented, and figure out which of the variables listed in [Styles.d.ts](https://github.com/samdenty/console-feed/blob/master/src/definitions/Styles.d.ts) it depends on. Another approach might be to change some CSS outside of console-feed.

**Comment by Ajaya1000** [comment]
Hey @catarak  I would like to take over this issue if no one is currently working on this

**Comment by Ajaya1000** [comment]
Hey, @catarak  I would like to take over the issue if no one is currently working on this.

**Comment by Ajaya1000** [comment]
The problem is in BASE_FONT_SIZE, and it breaks after a value of 14 on Chrome and 18 on  firefox. @catarak  Please suggest to me how to proceed with it.

**Comment by mattbesancon** [comment]
@Ajaya1000 how do you know the problem comes from BASE_FONT_SIZE?

**Comment by Ajaya1000** [comment]
@mattbesancon I tried to debug the style which is passed to the console component(Console-Feed) and after a point, I found out only the BASE_FONT_SIZE style attribute has an effect on the bug. This isn't a problem with the p5-js web editor library, this is the problem of the console-feed package which depends on the react-inspector package.

**Comment by catarak** [comment]
@Ajaya1000 have you opened an issue in the console-feed or react-inspector libraries?

**Comment by Ajaya1000** [comment]
@catarak yes. I have opened this issue in the console-feed and the react-inspector library. But no movement so far.

**Comment by catarak** [comment]
Thanks! Captured in https://github.com/samdenty/console-feed/issues/79.

**Comment by parteekcoder** [comment]
@raclim is this issue not related to p5web-editor

**Comment by achyutvardhan** [comment]
i m new to open source . is this issue resolved ?

**Comment by vraj8902** [comment]
@Ajaya1000 Hey, could you please tell me if this issue is resolved or not? 
  if not i would like to work on this issue

**Comment by vraj8902** [comment]
@catarak Can i work on this issue if not resolved ?

**Comment by ssai-manoj** [comment]
Hi @catarak, can i know if the issue is solved?

**Comment by Rahulhanje** [comment]
Hey @catarak may i know that is this issue resolved ?

**Comment by SushantBansal-tech** [comment]
hi @catarak can i still work on this issue

**Comment by yashodipmore** [comment]
Hi,
@catarak 

I found a bug where console.table does not display properly in the console window. This issue affects the way data is presented in tabular format, making it difficult to read structured outputs.

Issue reference: Similar behavior as documented in MDN Web Docs for console.table.

I’d like to investigate and work on this issue. Could you please assign it to me?

Looking forward to your response!

**Comment by Aayush6015** [comment]
can someone help me to generate this issue, I am unable to do console.table(grid) in the web console.
